src/components/FileTreeNode/styles.module.scss

```diff
@@ -40,6 +40,10 @@
   width: 100%;
 }
 
+.expandedFolderName {
+  margin-left: 0.375rem;
```

Comment:
This is a bit fragile because if someone were to change the base font-size [again](https://github.com/mozilla/addons-code-manager/issues/771) it would not be obvious that this part of the layout will break. 

It would be safer to use pixels here but adjusting the margin seems like an indirect way to do it. Did you look into the source of the icons to see if there's a more direct way to specify their dimensions?

Reply:
Setting the width of the icon (the `min-width` of the svg image) makes transition between open and closed folder icon a bit different.
- Fix the width of the icon: Since the icon is in the center of the svg image, the left edge of the icon shifted when open/close the folder. I have no way to set svg attribute `preserveAspectRatio` to align the icon to the left edge of the svg container. 
![icon](https://user-images.githubusercontent.com/4984681/62142850-4435b400-b2ac-11e9-989b-0abedcdafe29.gif)
- Adjusting margin only
![icon_f](https://user-images.githubusercontent.com/4984681/62142851-4435b400-b2ac-11e9-97e9-eee2b7d84fa8.gif)

I updated the patch to adjust the margin of icon, instead of the folder name. I guess it is a little better. Please let me know if you would like to fix the width of the icon.

Reply:
Now I changed the unit to `em` to be consistent with the unit of icon width which is defined in `fontawesome`, hoping it is fine :) .